cli/app.go

```diff
@@ -361,6 +377,15 @@ var app = &cli.App{
 			Aliases: []string{"q"},
 			Usage:   "suppress warnings",
 		},
+		&cli.StringFlag{
+			Name:  profileFlag,
+			Usage: "Specify a particular profile for the current command",
+		},
+		&cli.BoolFlag{
+			Name:    disableProfilesFlag,
+			Aliases: []string{"disable-profile"},
```

Comment:
Why add this alias?

Reply:
When I was testing I found myself waffling between the two and found it a little annoying when the CLI told me one was wrong, so I added the alias for simplicity of use. Definitely not necessary, happy to remove if you think it's unnecessary cluttering!

Reply:
Seems fine to keep; maybe just a comment above saying it's not for backward compatibility (as most of our aliases are) but for ease of use.